Signal/src/view controllers/UITests/SignalsViewController.m

```diff
@@ -324,10 +324,8 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
 
 - (void)presentThread:(TSThread *)thread keyboardOnViewAppearing:(BOOL)keyboardOnViewAppearing {
     dispatch_async(dispatch_get_main_queue(), ^{
-      if (!_mvc) {
-          _mvc = [[UIStoryboard storyboardWithName:@"Storyboard" bundle:NULL]
-              instantiateViewControllerWithIdentifier:@"MessagesViewController"];
-      }
+      _mvc = [[UIStoryboard storyboardWithName:@"Storyboard" bundle:NULL]
+                instantiateViewControllerWithIdentifier:@"MessagesViewController"];
```

Comment:
We can have a shared MessageViewController! Load the drafted message in viewDidAppear and clear the compose Textview on viewWillDissapear...